csm/cli/cortxcli.py

```diff
@@ -271,6 +273,7 @@ def do_exit(self, args=""):
 if __name__ == '__main__':
     cli_path = os.path.realpath(sys.argv[0])
     sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..'))
+    sys.path.append(os.path.join(os.path.dirname(pathlib.Path(cli_path)), '..', '..'))
```

Comment:
Appending a new path to the list of paths based on `sys.argv[0]` is prone to [resource injection vulnerabilities](https://owasp.org/www-community/attacks/Resource_Injection) as `sys.argv[0]` can be manipulated by an attacker by using [the `exec*` syscalls](https://manpage.me/?q=execv).  If we have any idea on where will this file be installed it would be safer to use the path directly or to figure it out in a different way.

This is especially troublesome in the case of CORTX CLI since it is supposed to be a multi-user application.

Reply:
Comment addressed.